### PR TITLE
Update the version of the v3 vsixes

### PR DESCRIPTION
**Pull request by basoundr**
Remove hard-coded version for Editors V3

@srivatsn @dotnet/project-system for review

### CODE REVIEW

**Comment by jinujoseph** [comment]
👍

**Comment by natidea** [comment]
Are we taking this to escrow or is it safe to hold for next release?

File: src/VsixV3/ProjectSystemPackage/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.swixproj [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <Import Project="$(NuGetPackageRoot)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.props" />
 
   <PropertyGroup>
-    <PackagePreprocessorDefinitions>$(PackagePreprocessorDefinitions);Version=$(AssemblyVersion);OutputPath=$(OutDir);SourcePath=$(OutDir)\..\..\src;LicenseUri=http://www.microsoft.com</PackagePreprocessorDefinitions>
+    <PackagePreprocessorDefinitions>$(PackagePreprocessorDefinitions);Version=$(BuildVersion);OutputPath=$(OutDir);SourcePath=$(OutDir)\..\..\src;LicenseUri=http://www.microsoft.com</PackagePreprocessorDefinitions>
```
Comment (natidea) [inline src/VsixV3/ProjectSystemPackage/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.swixproj]: Is BuildVersion greater than what is currently shipping? Does it pick the right scheme (2.0.0 vs 15.0.0) for CommonFiles vs Editors?

**Comment by natidea** [comment]
Do we also need to change Microsoft.VisualStudio.ProjectSystem.Managed.swixproj ?

**Comment by srivatsn** [comment]
Tagging @MattGertz for approval. The version number of the main projectsystem vsix is getting incremented properly but not the other two vsixes we produce. Those VSIXes don't change often but the fix we made yesterday to fix the project guid regression goes into this VSIX. Since the release branch is using incremental versioning, without this change, that fix doesn't take effect. Below is the justification of the bug that this fixes:

#### Customer scenario
Project Guids get stamped into the project. Many property pages show up disabled. Many other features that depend on getting project properties are broken. The reason is that because of a change in order of files being imported, our CPS rules for project properties weren't getting imported.
#### Bug this fixes
Fixes #681, Fixes #676 
#### Workarounds, if any
Users have to hand edit the target files which is risky
#### Risk
This change is creating a replica of all the common xaml rules in Roslyn Project System. So minimal risk.
#### Performance impact
None
#### Is this a regression from a previous update?
Yes
#### Root cause analysis:  
A recent change that we made to change the order of importing targets files in MSBuild caused this regression. We need integration tests to catch these issues.
#### How was the bug found?
Ad hoc testing

File: src/VsixV3/ProjectSystemPackage/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.swixproj (hunk 1 above)
Comment (basoundr) [inline src/VsixV3/ProjectSystemPackage/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.swixproj]: Yes, this build number will be constructed out of the build number which will be greater than the previous versions all the time

**Comment by basoundr** [comment]
@natidea fixed  Microsoft.VisualStudio.ProjectSystem.Managed.swixproj as well

**Pull request merged**